internal/faustjs.org/docs/gutenberg/wp-graphql-content-blocks.mdx

```diff
@@ -253,7 +253,7 @@ This way you can reconstruct the block tree as before and pass it on to the `Wor
 
 ```js
 import { WordPressBlocksViewer } from '@faustwp/blocks';
-import flatListToHierarchical from '../utilities/flatListToHierarchical';
+import { flatListToHierarchical } from '@faustwp/core/src';
```

Comment:
We will want users to be able to import this function directly from @faustwp/core (without the need for /src)